Application/home.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Form Application</title>
+    <link rel="stylesheet" href="form.css">
```

Comment:
I was not able to find the file `form.css`. If this is not needed this line should be removed

Reply:
Last time i changed the file name but i didn't changed in here. It's actually registerationForm.css file.